clip_image_classifier.py

```diff
@@ -54,7 +62,7 @@ def classify(self, docs: DocumentArray, parameters: dict, **kwargs):
                 'or in constructor. e.g. ["This is a cat","This is a person", "This is a dog"]'
             )
         for docs_batch in docs[
-            parameters.get('traversal_paths', self.traversal_paths)].batch(
+            parameters.get('access_paths', self.access_paths)].batch(
```

Comment:
Please add warning when `traversal_paths` is used. FYI, https://github.com/jina-ai/executor-text-dpr-encoder/pull/7/files#